src/fw/drivers/vibe_drv2604.c

```diff
@@ -40,30 +45,117 @@ void vibe_init(void) {
   uint8_t rv;
   bool found = prv_read_register(0x00, &rv);
   if (found) {
-    PBL_LOG(LOG_LEVEL_DEBUG, "Found DRV2604 with STATUS register %02x", rv);
+    PBL_LOG(LOG_LEVEL_INFO, "Found DRV2604 with STATUS register %02x", rv);
   } else {
     PBL_LOG(LOG_LEVEL_ERROR, "Failed to read the STATUS register");
   }
+  
+  /* calibration table maybe should live in the board file? */
+  bool bad = false;
+  bad |= !prv_write_register(0x01, 0x00); /* exit standby */
+  bad |= !prv_write_register(0x1A, 0xAA); /* ERM_LRA | FB_BRAKE_FACTOR = 2 | LOOP_GAIN = 2 | BEMF_GAIN = 2 */
+  bad |= !prv_write_register(0x16, 0x3F); /* RATED_VOLTAGE set to default of 0x3F */
+  bad |= !prv_write_register(0x17, 0x89); /* OD_CLAMP set to default of 0x89 */
+  bad |= !prv_write_register(0x18, 0x0D); /* A_CAL_COMP = 0x0D */
+  bad |= !prv_write_register(0x19, 0x80); /* A_CAL_BEMF = 0x80 */
+  bad |= !prv_write_register(0x1E, 0x30); /* AUTO_CAL_TIME = 3 */
+  bad |= !prv_write_register(0x1B, 0x90); /* STARTUP_BOOST | DRIVE_TIME = 2.1 ms = 0x10 */
+  bad |= !prv_write_register(0x1C, 0xF5); /* BIDIR_INPUT | BRAKE_STABILIZER | SAMPLE_TIME = 3 | BLANKING_TIME = 1 | IDISS_TIME = 1 */
```

Comment:
Add defines, so code is autoreadable

For error handling: we should return on every error; otherwise we can have undefined behavior on the target device.

Reply:
Indeed.  Have another look?